### PR TITLE
system_tests: use requirements.txt from both libbeat and beater 

### PR DESCRIPTION
**Pull request by cyrilleverrier**
Community beats can add `./tests/system/requirements.txt` to install additional Python libraries, in addition to `libbeat/tests/system/requirements.txt`

### CODE REVIEW

**Comment by cyrilleverrier** [comment]
@ruflin if this PR is accepted, please add a dummy requirements.txt to your cookiecutter project as an example.

**Comment by ruflin** [comment]
Interesting. Will have a closer look later. This is somehow also related to https://github.com/elastic/beats/pull/904

File: libbeat/tests/system/requirements.txt [hunk 1]
```diff
@@ -2,3 +2,5 @@ nose
 jinja2
 PyYAML
 nose-timer
+pip
```
Comment (ruflin) [inline libbeat/tests/system/requirements.txt]: Can pip be a requirement of pip?

Comment (cyrilleverrier) [inline libbeat/tests/system/requirements.txt]: Yes , il will update the outdated version of pip which is shipped with virtualenv

Comment (ruflin) [inline libbeat/tests/system/requirements.txt]: Nice

**Comment by ruflin** [comment]
Thx, merged. Will add a requirements.txt file to beat-generator.

File: libbeat/tests/system/requirements.txt (hunk 1 above)
Comment (cyrilleverrier) [inline libbeat/tests/system/requirements.txt]: @ruflin: Thank you

**Pull request merged**